app-backend/db/src/main/scala/util/database.scala

```diff
@@ -50,6 +50,7 @@ object RFTransactor {
       hikariConfig.setUsername(user)
       hikariConfig.setPassword(password)
       hikariConfig.setDriverClassName(driver)
+      hikariConfig.setConnectionTimeout(2000)
```

Comment:
Calling this out specifically - it's a change from the default (30 seconds) which would have been way too long. This lets us identify real failures to connect more quickly rather than getting a generic timeout in the healthcheck.